source/system/MyButtercupAdapter.js

```diff
@@ -0,0 +1,22 @@
+"use strict";
+
+const fetch = require("node-fetch");
+const archiveTools = require("../tools/myButtercup/archive");
+
+module.exports = function buildAdapter(archiveID, accessToken) {
+    return {
+
+        getArchiveData: function getArchiveData() {
```

Comment:
We can of course, but I don't want to break old nodes instantly with this release, even though we've stopped support.